src/main/java/com/google/firebase/remoteconfig/internal/TemplateResponse.java

```diff
@@ -108,29 +94,28 @@ public RemoteConfigParameter toRemoteConfigParameter() {
     private String value;
 
     @Key("useInAppDefault")
-    private Boolean inAppDefaultValue;
+    private Boolean useInAppDefault;
 
-    public ParameterValueResponse() {
+    public String getValue() {
+      return value;
     }
 
-    private ParameterValueResponse(String value, Boolean inAppDefaultValue) {
-      this.value = value;
-      this.inAppDefaultValue = inAppDefaultValue;
+    public Boolean getUseInAppDefault() {
```

Comment:
Is this used anywhere? If not lets remove it. And also change the setter to accept `boolean` so it matches the getter.

Reply:
Good call. Updated!